classes/user/form/IdentityForm.php

```diff
@@ -123,14 +124,15 @@ public function execute(...$functionArgs)
 
 
         // Request to delete ORCID token is handled separately from other form field updates
-        if($this->getData('removeOrcidId') === 'true') {
+        if ($this->getData('removeOrcidId') === 'true') {
             $user->setOrcid(null);
             $user->setOrcidVerified(false);
             OrcidManager::removeOrcidAccessToken($user);
         } else {
             $user->setGivenName($this->getData('givenName'), null);
             $user->setFamilyName($this->getData('familyName'), null);
             $user->setPreferredPublicName($this->getData('preferredPublicName'), null);
+            $user->setPreferredAvatarInitials($this->getData('preferredAvatarInitials'), null);
```

Comment:
Should these be transformed to capital letters at some point? Maybe enforced at the time of typing or here? Or should it only happen in the avatar component. What do you think?

Reply:
Good question! I think it makes sense to capitalize it at the input stage. This would keep it consistent with how it's eventually shown to user

Reply:
I decided to handle the capitalization in both places. It's better for UX to show the change as the user types, and I prefer not to rely solely on the front-end for the final data format(in case a user populates the field with JS, etc ).